apps/onlyoffice.sh

```diff
@@ -20,32 +17,60 @@ debug_mode
 # Check if root
 root_check
 
-# Nextcloud 13 is required.
-lowest_compatible_nc 13
+# Nextcloud 18 is required.
+lowest_compatible_nc 18
 
 # Test RAM size (2GB min) + CPUs (min 2)
 ram_check 2 OnlyOffice
 cpu_check 2 OnlyOffice
 
-# Notification
-msg_box "Before you start, please make sure that port 80+443 is directly forwarded to this machine!"
-
 # Get the latest packages
 apt update -q4 & spinner_loading
 
-# Check if Nextcloud is installed
-print_text_in_color "$ICyan" "Checking if Nextcloud is installed..."
-if ! curl -s https://"${NCDOMAIN//\\/}"/status.php | grep -q 'installed":true'
+# Check if onlyoffice is already installed
+print_text_in_color "$ICyan" "Checking if Onlyoffice is already installed..."
+if version_gt "$CURRENTVERSION" "18.0.1" && ! does_this_docker_exist 'onlyoffice/documentserver'
```

Comment:
Actually, if you don't delete the ! before does_this_docker_exist, it will directly exit the if-then here.